nsot/models.py

```diff
@@ -28,7 +28,7 @@
 # These are constants that becuase they are tied directly to the underlying
 # objects are explicitly NOT USER CONFIGURABLE.
 RESOURCE_BY_IDX = (
-    'Site', 'Network', 'Attribute', 'Device', 'Interface'
+    'Site', 'Network', 'Attribute', 'Device', 'Interface', 'Circuit'
```

Comment:
Would look cleaner to come from a separate diff, but yeah agreed. It's not a huge thing would would safe a couple bits of redundancy.